### PR TITLE
Fixed the problem for issue #320. 

### PR DESCRIPTION
**Pull request by gerlou**

### CODE REVIEW

File: editor/static/js/editor.js [hunk 1]
```diff
@@ -415,9 +415,8 @@ $(document).ready(function() {
         this.setTab = function(id) {
             return function() {
                 var tab = ei.getTab(id);
-
                 ei.currentTab(tab);
-                ei.set_ignored_publishing_criteria(false);
+                ei.ignored_publishing_criteria(false);
```
Comment (christianp) [inline editor/static/js/editor.js]: This doesn't work - most of the time, the tab is changed by modifying the `currentTab` observable directly. 
Anyway, this is not the right place to add a side-effect which isn't part of the tab system - instead, there should be a computed observable whose only job is to watch the current tab and set `ignored_publishing_criteria` if the tab changes.

**Comment by christianp** [comment]
I don't think it's necessary for the "ignore" status to reset when you leave the tab - no other element of the interface resets when you change tabs.

The "ignore" button should disappear after you click it.

Now you can have two "publish" panels visible at once, like so:
image

The "Publish" button should replace the "Ignore" button when you click it, or the "ignore" button should change to "click again to confirm", so you have to click it twice.

**Comment by christianp** [comment]
(Thanks for sticking with this, by the way! It's turning out to be a bit more complicated than I imagined)

**Comment by gerlou** [comment]
Hi Christian,

I'll change it so when you click on the ignore button, the Ignore button itself will disappear and then the Publish panel will show up.

**Comment by gerlou** [comment]
I just modified the code and added some commits to the pull request.

**Comment by christianp** [comment]
Thanks for this. I've changed your code a bit, so you just have to click the "ignore" button twice rather than move to the normal "Publish" button - in your version, when both panels were visible there was some repeated text.
I squashed all your commits and mine into one new commit, so it looks like GitHub didn't close this automatically. 
Thanks for adding a feature to Numbas!

**Pull request closed without merging**